lib/containers/basetest.pm

```diff
@@ -38,4 +40,20 @@ sub containers_factory {
     return $engine;
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    return if get_var('NOLOGS');
+
+    select_console('log-console');
```

Comment:
This line here, is breaking some container tests tumbleweed: https://openqa.opensuse.org/tests/5162233#step/image_docker/227

https://openqa.opensuse.org/tests/overview?arch=&flavor=&machine=&test=&modules=&module_re=&group_glob=&not_group_glob=&comment=&todo=1&distri=microos&distri=opensuse&version=Tumbleweed&build=20250709&groupid=1#